src/main/java/de/rytrox/varo/discord/MessageService.java

```diff
@@ -128,7 +126,7 @@ public void writeMessage(String message, DiscordColor color, boolean addTimestam
 
             } catch (IOException ex) {
                 Bukkit.getConsoleSender().sendMessage("ERROR - Discord-Nachricht konnte nicht gesendet werden");
```

Comment:
Hier kannst du eher auf den Logger zurückgreifen. Den Findest du in der Mainklasse unter JavaPlugin#getLogger()